src/integrations/prefect-docker/prefect_docker/deployments/steps.py

```diff
@@ -221,6 +238,18 @@ def build_docker_image(
         with Path(temp_dockerfile).open("w") as f:
             f.writelines(line + "\n" for line in lines)
 
+        if persist_dockerfile:
+            logger.info(
+                "Persisting auto-generated Dockerfile to disk. You can use `dockerfile_output_path` to specify the output path."
```

Comment:
This is closer to the log message I had in mind:
```suggestion
                f"Persisting auto-generated Dockerfile to {dockerfile_output_path or 'Dockerfile.generated'}. Please update your 'dockerfile' value to use this Dockerfile for subsequent runs."
```